Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -226,7 +226,10 @@ extension ChannelDTO {
 extension _ChatChannel {
     /// Create a ChannelModel struct from its DTO
     fileprivate static func create(fromDTO dto: ChannelDTO) -> _ChatChannel {
-        let members: [_ChatChannelMember<ExtraData.User>] = dto.members.map { $0.asModel() }
+        let memberCompare: (MemberDTO, MemberDTO) -> Bool = { lhs, rhs in
+            lhs.user.lastActivityAt ?? Date(timeIntervalSince1970: 0) < rhs.user.lastActivityAt ?? Date(timeIntervalSince1970: 0)
+        }
+        let members: [_ChatChannelMember<ExtraData.User>] = dto.members.sorted(by: memberCompare).prefix(25).map { $0.asModel() }
```

Comment:
Are you sure about this?
```
dto.members.sorted(by: memberCompare).prefix(25)
```
There can be thousands of members in the channel. I don't think it's a good idea to iterate over all of them. Maybe it would be better to create a fetch request for this? You can also specify the sorting there directly.

Reply:
Well that's why I've been thinking about fetched property, which is basically a fetch request...and then prefixing that, do you have any preference?

Reply:
Solely for the purpose of having better control over it, I think a fetch request is better for this